public/components/metrics/sidebar/metrics_accordion.tsx

```diff
@@ -11,10 +11,11 @@ interface IMetricNameProps {
   metricsList: [];
   headerName: string;
   handleClick: (props: any) => void;
+  dataTestSubj: string;
 }
 
 export const MetricsAccordion = (props: IMetricNameProps) => {
-  const { metricsList, headerName, handleClick } = props;
+  const { metricsList, headerName, handleClick, dataTestSubj } = props;
```

Comment:
Why is this being passed in props? Can't it be declared here in this file?

Reply:
It can't be declared in that file as the same function is being used to list all of selected metrics, recently created and also available metrics. Won't be able to differentiate between the three for testing